ui-participant/src/landing/registration/Preregistration.tsx

```diff
@@ -19,13 +19,12 @@ export default function PreRegistration({ registrationContext }: { registrationC
     if (!surveyModel) {
       return
     }
-    const responseDto = {
+    const responseDto: Partial<PreregistrationResponse> = {
       resumeData: getResumeData(surveyModel, null),
       answers: getSurveyJsAnswerList(surveyModel),
-      creatingParticipantId: null,
```

Comment:
This isn't a field of pre-registration responses. There is a `creatingParticipantUserId` field.